spigot-tests/index.js

```diff
@@ -33,6 +34,8 @@ const config = {
     username: envConfig.USERNAME,
     password: envConfig.PASSWORD
   },
+  appName: 'webrtc-sdk',
```

Comment:
Is this the same as `logTopic` found in [logging.ts](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/blob/master/src/logging.ts#L38)?

Reply:
kind of. it uses this, but spigot prefixes this with `spigot-` so we'll easily know what is what.

Reply:
Perfect. That's what I was hoping for 👍